config/query/format_boosting.yml

```diff
@@ -0,0 +1,49 @@
+government_index:
```

Comment:
I think whatever we do we should assume people will forget/not know to update this file when adding a new format.

So for this work (not necessarily in this PR, but we should have a plan before deploying it) we should make sure that unrecognised formats either get boosted down by default or alert us in some way.

There's a master list here we might be able to use if we opt for the latter: https://github.com/alphagov/govuk-content-schemas/blob/master/lib/govuk_content_schemas/allowed_document_types.yml